cc/algorithms/algorithm.h

```diff
@@ -195,6 +201,13 @@ class AlgorithmBuilder {
                    << " is being used. Consider setting your own epsilon based "
                       "on privacy considerations.";
     }
+
+    if (!delta_.has_value()) {
```

Comment:
I think we do not need to make this distinction.  Delta will be set as 0 by default and the numerical mechanism might fail (if it requires delta).

Reply:
Fixed in next commit.